.github/workflows/build.yml

```diff
@@ -1,42 +1,11 @@
 name: Build
```

Comment:
Maybe this could be re-named to _e2e-test_?

```suggestion
name: E2E-Test
```

... and the _image_ target could be separated as well?

Reply:
yes, this makes sense. I would actually even remove the `image` target, since it's part of e2e build anyway